cloud-data-protection/src/App.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {Layout} from './components/layout';
+import './App.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import "./App.css";
```